build/packaging/post_build_script.sh

```diff
@@ -7,4 +7,5 @@
 
 set -eux
 
+rm -rf pip-out/
```

Comment:
For debugging it would be be helpful to keep these files around. How about `mv pip-out BACKUP-pip-out` or similar?